lib/chewy/type/mapping.rb

```diff
@@ -173,6 +175,15 @@ def mappings_hash
           root_object ? root_object.mappings_hash : {}
         end
 
+        # Does the type have incremental_update_field field defined? And is this
```

Comment:
I would rather state it in simple sentences:
> Check whether a type has incremental_update_field set and that field is defined with a simple value.

Also, I'm still considering using `outdated_sync` instead of `increamental_update`. What do you think about that?